cli/utils.ts

```diff
@@ -42,14 +42,14 @@ export const filterAndSortFiles = (
   files: Deno.DirEntry[],
   queryResult: string | undefined,
 ): Deno.DirEntry[] => {
-  return files.filter((file: Deno.DirEntry): boolean => {
-    if (!regexFileName.test(file.name)) return false;
+  return files
+    .filter((file: Deno.DirEntry): boolean => {
+      if (!regexFileName.test(file.name)) return false;
 
-    if (queryResult === undefined) return true;
+      if (queryResult === undefined) return true;
 
-    return parseInt(file.name.split("-")[0]) >
-      new Date(queryResult).getTime();
-  })
+      return file.name > queryResult[0];
```

Comment:
```suggestion
      return file.name > queryResult;
```

Reply:
Is queryResult an array or a string? In PostgreSQL case it used to return an array hence the array index of 0. Should I commit this just to check?

Reply:
The query result will always be a string or undefined (see the type), the logic for parsing the arrays are done before this method is called